File: POOUber/python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMeterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMeterial = seatsMaterial

File: POOUber/php/uberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car{
    public $typeCarAccepted;
    public $seatMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatMaterial){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterial = $seatMaterial;
    }
}